bureaucracy.js

```diff
@@ -41,6 +42,9 @@ function create (options) {
     }
     bureaucrat.emit('valid', validFiles);
     var form = new FormData();
+    Object.keys(o.formData).forEach(function copyFormData(key) {
+      form[key] = o.formData[key];
```

Comment:
really? I get `form.add is not a function` in a console, but maybe I'm
doing it wrong?

On Thu, Aug 4, 2016 at 11:07 AM, Nicolás Bevacqua <notifications@github.com>
wrote:

> In bureaucracy.js
> <https://github.com/bevacqua/bureaucracy/pull/3#discussion_r73540227>:
>
> > @@ -41,6 +42,9 @@ function create (options) {
> >      }
> >      bureaucrat.emit('valid', validFiles);
> >      var form = new FormData();
> > +    Object.keys(o.formData).forEach(function copyFormData(key) {
> > +      form[key] = o.formData[key];
>
> form.add(key, o.formData[key])
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bevacqua/bureaucracy/pull/3/files/61e64d2c8f4f147580c78cda51f9c660c63b78f5#r73540227>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ8gc3BE6l2qt7qD1M2-2JlwZFh9zks5qcgAWgaJpZM4Jcyxq>
> .
>